index.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="Microverse portifolio clone activity.">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css"/>
+  <title>Microverse Portifolio - Template 3</title>
+</head>
+<body>
+  <header>
+    <div class="toolbar-container">
+      <div class="toolbar logo">
+        <a class="logo-link" href="#" title="Welcome HomePage link">
+            <h3 class="logo-text default-custom-font">Welcome</h3>
+        </a>
+      </div>
+      <nav class="toolbar menu">
+        <div class="menu-trigram">
+          <a class="menu-link" href="#" title="Open Menu link"></a>
+        </div>
+      </nav>
+    </div>
+  </header>
+  <section class="headline-container">
+    <div class="headline primary-text">
+      <h1 class="primary-text first-color primary-text-custom-font">Hey there. I'm Nelsino</h1>
+      <h2 class="primary-text second-color primary-text-custom-font">I'm a software developer</h2>
+    </div>
+    <div class="headline supporting-text">
+      <p class="sup-text-paragraph default-custom-font">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <ul class="headline social-media-icons">
+      <li>
+        <a class="twitter" href="https://twitter.com/nelsino_s" title="Contact-me: Twitter link"></a>
+      </li>
+      <li>
+        <a class="linkedin" href="https://www.linkedin.com/in/nelsino-francisco-de-souza-neto-ab60a7166/" title="Contact-me: Linkedin link"></a>
+      </li>
+      <li>
+        <a class="medium" href="#" title="Contact-me: Medium link"></a>
+      </li>
+      <li>
+        <a class="github" href="https://github.com/nelsinofrancisco" title="Contact-me: Github link"></a>
+      </li>
+      <li>
+        <a class="angellist" href="#" title="Contact-me: AngelList link"></a>
+      </li>
+    </ul>
+    <div class="headline action">
+      <a class="action-btn" href="#" title="Display next section">
+        <div class="chevron"></div>
+      </a>
+    </div>
+  </section>
+  <section class="recent-works-container">
+    <div class="recent-works intro-header">
+      <div class="intro-header primary-text">
+        <h2 class="primary-text first-color primary-text-custom-font">My Recent Works</h2>
+      </div>
+      <div class="intro-header indicator">  
+      </div>
+    </div>
+    <div class="card-work-container">
+      <div class="card-work card-1">
```

Comment:
Example for the _[OPTIONAL CHANGE]_

That div and its peers could be replaced with an `<article>` tag.